server/schedulers/balance_test.go

```diff
@@ -365,22 +365,17 @@ func (s *testBalanceRegionSchedulerSuite) TestReplicas3(c *C) {
 	tc.addLabelsStore(6, 1, map[string]string{"zone": "z1", "rack": "r1", "host": "h1"})
 	CheckTransferPeer(c, sb.Schedule(tc, schedule.NewOpInfluence(nil, tc)), schedule.OpBalance, 1, 6)
 
-	// Store 7 has the same region score with store 6, but in a different host.
+	// Store 7 has the same region score with store 6.
 	tc.addLabelsStore(7, 1, map[string]string{"zone": "z1", "rack": "r1", "host": "h2"})
-	CheckTransferPeer(c, sb.Schedule(tc, schedule.NewOpInfluence(nil, tc)), schedule.OpBalance, 1, 7)
+	CheckTransferPeerFrom(c, sb.Schedule(tc, schedule.NewOpInfluence(nil, tc)), schedule.OpBalance, 1)
```

Comment:
How about assign a smaller score to store7 (less than store6) here? So we can be sure the transfer target will be 7?